pertpy/metadata/_cell_line.py

```diff
@@ -294,6 +297,9 @@ def annotate_cell_lines(
                 # We will subset the original metadata dataframe correspondingly and add them to the AnnData object.
                 # Again, redundant information will be removed.
                 if reference_id not in cell_line_information:
+                    # Sequence doesnt have append function, turn into list
```

Comment:
Well, in that case, we could actually require it to be `list`. I didn't see this ufff. You can change it now, but you don't have to